### DISCUSSION TITLE
crazyflie flies unstable

### DISCUSSION THREAD

**Discussion by JaouaherBelgacem** [bitcraze/discussions#840] [Q&A]
Hi, my crazyflie 2.1 is flying like crazy and endup crashing in the ground is there any ways to make flying in one position in a stable way?
Thanks

**Comment by krichardsson** [comment]
Hi, 
could you describe your set up a bit? Do you use any decks, are you flying manually or using scripts? Maybe add a video if you can

**Reply by JaouaherBelgacem** [reply]
yes here is a video 

https://github.com/bitcraze/discussions/assets/91470118/a32722ea-e290-4cd0-8931-cfee99511eea

**Reply by krichardsson** [reply]
But are you flying manually or with a script? 
If you are using a script, what is it doing? 
Are you using the lighthouse for positioning? Is it set up properly?

**Reply by JaouaherBelgacem** [reply]
Yes I am using the lighthouse for positioning and I am flying manually

**Reply by krichardsson** [reply]
First step would be to remove the lighthouse deck (and any other decks) and try to fly manually (with a gamepad) to make sure that works and that the Crazyflie is working as expected.

Secondly make sure the lighthouse system is set up properly (#839). Flying with a lighthouse system that is not OK will push weird data in the to estimator which makes it very confused. My guess is that this is the root cause of your fly-aways, but I'm not sure.
